### PR TITLE
Fix config setting bug and add ability to get listener

### PR DESCRIPTION
**Pull request by MatthewScholefield**
Previously, the config assignment was bypassed when the code returned early to speak to the user that the download has started.

This also allows for phrases like `what is the current listener`.

### CODE REVIEW

File: vocab/en-us/GetKeyword.voc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+whats
```
Comment (penrods) [inline vocab/en-us/GetKeyword.voc]: normalize() takes care of this, transforming "what's" and "whats" into "what is"

File: vocab/en-us/GetKeyword.voc [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+what is
+give me
+show me
```
Comment (penrods) [inline vocab/en-us/GetKeyword.voc]: "Show" is an odd word here.  In the future (18.02) we will be using "show me" to indicate things that will be sent to an associated screen.  If you would intend this to show something, then I'm fine leaving this.

We should allow "tell me"

**Review by penrods — CHANGES_REQUESTED** [review]
Couple minor things, then good to go.

**Comment by MatthewScholefield** [comment]
OK, I've amended the commit.

**Review by penrods — APPROVED** [review]
LGTM

**Pull request merged**